### PR TITLE
fix(coderd): check if timings can be read

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

**Review by mtojek — APPROVED** [review]
Screenshot 2024-09-17 at 14 52 33

File: coderd/database/dbauthz/dbauthz.go [hunk 1]
```diff
@@ -1792,6 +1792,10 @@ func (q *querier) GetProvisionerJobByID(ctx context.Context, id uuid.UUID) (data
 }
 
 func (q *querier) GetProvisionerJobTimingsByJobID(ctx context.Context, jobID uuid.UUID) ([]database.ProvisionerJobTiming, error) {
+	_, err := q.GetProvisionerJobByID(ctx, jobID)
```
Comment (dannykopping) [inline coderd/database/dbauthz/dbauthz.go:1795]: This approach raised my eyebrow, and I was going to suggest not doing it this way but I see we have precedent for this in `GetProvisionerJobByID` itself. Please add a comment explaining this.

**Review by dannykopping — APPROVED** [review]

File: coderd/database/dbauthz/dbauthz.go (hunk 1 above)
Comment (mtojek) [reply coderd/database/dbauthz/dbauthz.go:1795]: I'm sure we use a similar pattern for workspace builds or templates.

**Comment by BrunoQuaresma** [comment]
> Screenshot 2024-09-17 at 14 52 33

LOL, but in my defense, the title is pretty descriptive and it is just 5 lines of code change

**Pull request merged**